docs/src/components/about/Team.tsx

```diff
@@ -0,0 +1,392 @@
+    locationCountry: 'br',
+    src: 'https://avatars.githubusercontent.com/u/29267813',
+  },
+  {
+    name: 'Danica Shen',
+    github: 'DDDDDanica',
+    title: 'Chinese docs',
+    location: 'Ireland',
+    locationCountry: 'ie',
+    src: 'https://avatars.githubusercontent.com/u/12678455',
+  },
+];
+
+const emeriti = [
+  {
+    name: 'Hai Nguyen',
+    github: 'hai-cea',
+    twitter: 'haicea',
+    title: 'MUI Core, v0.x creator',
+    location: 'Dallas, US',
+    locationCountry: 'us',
+    src: 'https://avatars.githubusercontent.com/u/2007468',
+  },
+  {
+    name: 'Nathan Marks',
+    github: 'nathanmarks',
+    title: 'MUI Core, v1.x co-creator',
+    location: 'Toronto, CA',
+    locationCountry: 'ca',
+    src: 'https://avatars.githubusercontent.com/u/4420103',
+  },
+  {
+    name: 'Kevin Ross',
+    github: 'rosskevin',
+    twitter: 'rosskevin',
+    title: 'MUI Core, flow',
+    location: 'Franklin, US',
+    locationCountry: 'us',
+    src: 'https://avatars.githubusercontent.com/u/136564',
+  },
+  {
+    name: 'Sebastian Sebald',
+    github: 'sebald',
+    twitter: 'sebastiansebald',
+    title: 'MUI Core',
+    location: 'Freiburg, Germany',
+    locationCountry: 'de',
+    src: 'https://avatars.githubusercontent.com/u/985701',
+  },
+  {
+    name: 'Ken Gregory',
+    github: 'kgregory',
+    title: 'MUI Core',
+    location: 'New Jersey, US',
+    locationCountry: 'us',
+    src: 'https://avatars.githubusercontent.com/u/3155127',
+  },
+  {
+    name: 'Tom Crockett',
+    github: 'pelotom',
+    twitter: 'pelotom',
+    title: 'MUI Core',
+    location: 'Los Angeles, US',
+    locationCountry: 'us',
+    src: 'https://avatars.githubusercontent.com/u/128019',
+  },
+  {
+    name: 'Maik Marschner',
+    github: 'leMaik',
+    twitter: 'leMaikOfficial',
+    title: 'MUI Core',
+    location: 'Hannover, Germany',
+    locationCountry: 'de',
+    src: 'https://avatars.githubusercontent.com/u/5544859',
+  },
+  {
+    name: 'Oleg Slobodskoi',
+    github: 'kof',
+    twitter: 'oleg008',
+    title: 'MUI Core, JSS',
+    location: 'Berlin, Germany',
+    locationCountry: 'de',
+    src: 'https://avatars.githubusercontent.com/u/52824',
+  },
+  {
+    name: 'Dmitriy Kovalenko',
+    github: 'dmtrKovalenko',
+    twitter: 'goose_plus_plus',
+    title: 'MUI X, date pickers',
+    location: 'Kharkiv, Ukraine',
+    locationCountry: 'ua',
+    src: 'https://avatars.githubusercontent.com/u/16926049',
+  },
+  {
+    name: 'Josh Wooding',
+    github: 'joshwooding',
+    twitter: 'JoshWooding_',
+    title: 'MUI Core, J.P. Morgan',
+    location: 'London, UK',
+    locationCountry: 'gb',
+    src: 'https://avatars.githubusercontent.com/u/12938082',
+  },
+];
+
+export default function Team() {
+  return (
+    <React.Fragment>
+      <Section cozy>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+          }}
+        >
+          <Typography variant="h2" mb={1}>
```

Comment:
So we can use anchor linking

```suggestion
          <Typography variant="h2" mb={1} id="muiers">
```